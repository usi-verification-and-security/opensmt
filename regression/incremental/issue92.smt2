(set-logic QF_LRA)
(declare-fun v6 () Bool)
(declare-fun v7 () Bool)
(declare-fun v13 () Bool)
(declare-fun v15 () Bool)
(assert (=> (=> v15 v6) (= v13 v7)))
(check-sat)
(check-sat)
(push 1)
(check-sat)
(pop 1)
(assert (= v15 (= v13 v7)))
(check-sat)
(push 1)
(check-sat)
(pop 1)
(assert (distinct true v15))
(push 1)
(check-sat)
(pop 1)
(push 1)
(check-sat)
(pop 1)
(push 1)
(check-sat)
(pop 1)
(push 1)
(check-sat)
(pop 1)
(check-sat)
