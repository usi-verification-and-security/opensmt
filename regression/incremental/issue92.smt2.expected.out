sat
sat
sat
sat
sat
unsat
unsat
unsat
unsat
unsat
